custom_model_runner/requirements.txt

```diff
@@ -25,3 +24,5 @@ packaging
 markupsafe
 pydantic
 datarobot-storage
+datarobot-mlops>=10.2.0  # Required for the 'set_api_spooler' with arugments
```

Comment:
Should we now update all the base env images with this version?
And don't pull mlops-agent jar: https://github.com/datarobot/datarobot-user-models/blob/master/docker/dropin_env_base/pom.xml#L41-L43 ?

Reply:
I consider removing drum at all from the base env (let's talk) and leave the dependency for the drop-in environments. 
In any case, currently, the drop-in envs has the constraint `datarobot-drum==1.15.0`, so it'll not be a problem to publish the new drum in PyPI